azext_iot/dps_sdk/device_provisioning_service_service_runtime_client.py

```diff
@@ -11,9 +15,8 @@
 from .version import VERSION
 from .operations.device_enrollment_operations import DeviceEnrollmentOperations
 from .operations.device_enrollment_group_operations import DeviceEnrollmentGroupOperations
-from .operations.registration_status_operations import RegistrationStatusOperations
+from .operations.registration_state_operations import RegistrationStateOperations
 from . import models
-from azext_iot._constants import VERSION as extver
```

Comment:
import for combining version for user agent

Reply:
change reverted